src/methods/network_info.rs

```diff
@@ -1,3 +1,27 @@
+//! Requests information about nodes on the network.
+//!
+//! Returns the current state of node network connections (active peers, transmitted data, received data, known producers, etc.).
```

Comment:
“received data” is contained within “transmitted data”. Also, I think we can rephrase to something of this format.

> Queries the current state of node network connections.
>
> This includes information about active peers, transmitted data, known producers, etc.